content/docs/iac/using-pulumi/extending-pulumi/use-terraform-module.md

```diff
@@ -0,0 +1,283 @@
+---
+title_tag: "Use a Terraform Module in Pulumi"
+meta_desc: "Learn how to use existing Terraform modules directly in your Pulumi programs."
+title: Use a Terraform Module in Pulumi
+h1: Use a Terraform Module in Pulumi
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    iac:
+        name: Use a Terraform Module
+        parent: iac-extending-pulumi
+        weight: 6
+---
+
+This guide will walk you through the process of using existing Terraform modules directly in your Pulumi programs, allowing you to leverage the vast Terraform module ecosystem.
+
+{{< notes type="info" >}}
+**Prerequisites:**
+
+- The [Pulumi CLI](/docs/install/)
+- One of Pulumi's [supported language runtimes](/docs/languages-sdks/) installed
+- Access to a cloud provider account for deployment (e.g., AWS, Azure, Google Cloud)
+
+{{< /notes >}}
+
+## Why Use Terraform Modules in Pulumi?
+
+Terraform has a mature ecosystem with thousands of modules available in the [Terraform Registry](https://registry.terraform.io/). These modules encapsulate well-tested infrastructure patterns that you might want to leverage in your Pulumi projects without having to rewrite them.
+
+Also, many Terraform users have created their own custom modules and would like to avoid re-writing them in order to use Pulumi. Using Terraform modules directly within Pulumi allows you to use Terraform and Pulumi side-by-side, enabling the two technologies to coexist, taking a slower gentler migration path from Terraform to Pulumi. This is especially powerful in larger organizations, where some teams prefer Pulumi's workflows, and others prefer to continue using Terraform.
+
+### Key benefits:
+
+- **Leverage Existing Modules**: Use the rich ecosystem of Terraform modules directly in Pulumi.
+- **Migrate Gradually**: Incrementally migrate from Terraform to Pulumi without rewriting everything at once.
+- **Consistency**: Maintain consistency across teams that may be using a mix of Terraform and Pulumi.
+
+## How It Works
+
+Pulumi's Terraform Module provider allows you to consume Terraform modules as if they were native Pulumi packages. It works by:
+
+1. Automatically installing and managing [OpenTofu](https://opentofu.org/) (an open-source Terraform-compatible implementation) to execute the module.
+2. Translating Pulumi resource declarations to Terraform configurations.
+3. Managing state through your standard Pulumi state backend.
+4. Exposing module outputs as native Pulumi outputs.
+
+## Getting Started
```

Comment:
We have an emerging `pulumi convert` capability that lets you target the modules. It's coming a little hot though so we can possibly do a quick addition  to the docs after this PR.